examples/simple_hpc_pbs_peer_network/config.json

```diff
@@ -0,0 +1,118 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+        "location": "southcentralus",
+        "resource_group": "<NOT-SET>",
+        "vm_type": "Standard_HB60rs",
+        "compute_instances": 2,
+        "vnet_resource_group": "variables.resource_group",
+        "peer_network_resource_group": "",
+        "peer_vnet_name": ""
+    },
+    "vnet": {
+        "resource_group": "variables.vnet_resource_group",
+        "name": "hpcvnet",
+        "address_prefix": "10.2.0.0/20",
+        "subnets": {
+            "compute": "10.2.4.0/22"
+        },
+        "peer": {
+            "Network1": {
+                "resource_group": "variables.peer_network_resource_group",
+                "vnet_name": "variables.peer_vnet_name"
+            }
+        }
+    },
+    "resources": {
+        "headnode": {
+            "type": "vm",
+            "vm_type": "variables.vm_type",
```

Comment:
Should headnode be a cheap node like D8s_v3?

Reply:
Resolved